src/components/GenreSelect/Tests/GenreSelect.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import GenreSelect, { Genre } from "../Component/GenreSelect";
+
+const genres: Genre[] = ["ALL", "DOCUMENTARY", "COMEDY", "HORROR", "CRIME"];
```

Comment:
This array looks like duplicated in the <App/>

Reply:
Could you explain what is wrong?

Reply:
This Values are going to be used in several places in the application. 
At the moment it is uset twise: in the test and in the App component (provided as prop for GenreSelect)
Consider creating the constant and reuse it